sql/core/benchmarks/FunctionBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,32 @@
+OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Mac OS X 10.16
+Intel(R) Core(TM) i9-10910 CPU @ 3.60GHz
+scalar function (long + long) -> long/notnull wholestage on:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                27292          29059        1913         18.3          54.6       1.0X
+with long_add_magic                                                   7049           7070          20         70.9          14.1       3.9X
+with long_add_static_magic                                            6752           6791          67         74.1          13.5       4.0X
+
+OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Mac OS X 10.16
+Intel(R) Core(TM) i9-10910 CPU @ 3.60GHz
+scalar function (long + long) -> long/notnull wholestage off:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+--------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                 28601          28964         496         17.5          57.2       1.0X
+with long_add_magic                                                    6986           7156         150         71.6          14.0       4.1X
+with long_add_static_magic                                             6509           6539          32         76.8          13.0       4.4X
+
+OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Mac OS X 10.16
+Intel(R) Core(TM) i9-10910 CPU @ 3.60GHz
+scalar function (long + long) -> long/nullable wholestage on:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+--------------------------------------------------------------------------------------------------------------------------------------------
+with long_add_default                                                 34579          34627          56         14.5          69.2       1.0X
+with long_add_magic                                                    6783           6799          17         73.7          13.6       5.1X
+with long_add_static_magic                                             7220           7476         332         69.3          14.4       4.8X
```

Comment:
This may due to `StaticInvoke` adds boxing even for primitive return types. Will look into it.

Reply:
Yes, it looks like a regression. It would be great to have a better result consistently.

Reply:
Filed [SPARK-35281](https://issues.apache.org/jira/browse/SPARK-35281) for this.